WorkflowCombine/Sources/Worker.swift

```diff
@@ -28,7 +28,7 @@ import Workflow
 /// If there is, and if the workers are 'equivalent', the context leaves the existing worker running.
 ///
 /// If there is not an existing worker of this type, the context will kick off the new worker (via `run`).
-public protocol Worker: AnyWorkflowConvertible where Rendering == Void {
+public protocol Worker<Output>: AnyWorkflowConvertible where Rendering == Void {
     /// The type of output events returned by this worker.
     associatedtype Output
     associatedtype WorkerPublisher: Publisher where
```

Comment:
i assume we don't want this one to also be a primary associated type, but do correct me if there's some use case in which that would be valuable that i'm overlooking